magicblock-config/src/ledger.rs

```diff
@@ -171,32 +170,48 @@ pub enum LedgerResumeStrategyType {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum LedgerResumeStrategy {
     /// Reset the ledger and optionally the accountsdb.
-    /// (slot, remove_accounts)
-    Reset(u64, bool),
+    Reset { slot: u64, keep_accounts: bool },
     /// Resume from the last slot found in the ledger.
-    /// (replay)
-    Resume(bool),
+    Resume { replay: bool },
```

Comment:
Thanks for applying a similar idea regarding enum variants here.
Nice! this is so much better :)